addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -363,23 +363,15 @@ export class WysiwygAdapterComponent extends ComponentAdapter {
                  '[data-oe-type="monetary"], [data-oe-many2one-id], [data-oe-field="arch"]:empty')
             .filter((_, el) => {
                 return !$(el).closest('.o_not_editable').length;
-            });
-
-        // TODO review in master. This stable fix restores the possibility to
-        // edit the company team snippet images on subsequent editions. Indeed
-        // this badly relies on the contenteditable="true" attribute being on
-        // those images but it is rightfully lost after the first save.
-        // grep: COMPANY_TEAM_CONTENTEDITABLE
-        let $extraEditableZones = $editableSavableZones.find('.s_company_team .o_not_editable img');
-
-        // To make sure the selection remains bounded to the active tab,
-        // each tab is made non editable while keeping its nested
-        // oe_structure editable. This avoids having a selection range span
-        // over all further inactive tabs when using Chrome.
-        // grep: .s_tabs
-        $extraEditableZones = $extraEditableZones.add($editableSavableZones.find('.tab-pane > .oe_structure'));
+            })
+            // To make sure the selection remains bounded to the active tab,
+            // each tab is made non editable while keeping its nested
+            // oe_structure editable. This avoids having a selection range span
+            // over all further inactive tabs when using Chrome.
+            // grep: .s_tabs
+            .add($savableZones.find('.tab-pane > .oe_structure'));
```

Comment:
If you can leave the `$extraEditableZones` abstraction instead of saving this inside `$editableSavableZones` while those are not "savableZones", I would prefer. You would also avoid touching line you don't have to touch :wink:

Reply:
Let me know if you like it with my latest changes @qsm-odoo .